src/materials/DamagePlasticityStressUpdate.C

```diff
@@ -135,15 +130,6 @@ DamagePlasticityStressUpdate::initQpStatefulProperties()
       used, the following commented lines show several different options. Some other options are
       still being considered. In this code, we define the element length as the cube root of the
       element volume */
-
```

Comment:
This comment below is still worth keeping around in my opinion since this hasn't been resolved.